contributors.json

```diff
@@ -284,13 +284,7 @@
   "dipanshubhola1009": {
     "country": "India",
     "name": "Dipanshu Bhola"
-  },
-  "Divakar": {
-    "country": "India",
-    "Github": "https://github.com/rexdivakar",
-    "linkedin": "https://www.linkedin.com/in/divakar-r-9b34b86b/",
-    "name": "Divakar"
-  },
+  },  
```

Comment:
Looks like you deleted a lot of peoples data. Please revert it or start it from a fresh branch